staging/src/k8s.io/mount-utils/mount_linux_test.go

```diff
@@ -874,3 +876,130 @@ func makeFakeCommandAction(stdout string, err error, cmdFn func()) testexec.Fake
 		return testexec.InitFakeCmd(&c, cmd, args...)
 	}
 }
+
+func makeLink(link, target string) error {
+	if err := os.Symlink(target, link); err != nil {
+		return fmt.Errorf("mklink failed: %w, link(%q) target(%q)", err, link, target)
+	}
+	return nil
+}
+
+func removeLink(link string) error {
+	if err := os.Remove(link); err != nil {
+		return fmt.Errorf("rmlink failed: %w", err)
+	}
+	return nil
+}
```

Comment:
I meant use `os.Symlink` / `Remove` instead of `makeLink` / `removeLink`, these functions do not add any value.

Reply:
Sure, replace them